### ISSUE TITLE
local host name 

### ISSUE THREAD

**Issue by saun4app** [DockStation/dockstation#54]
Are we able to use` DockStation` to setup a local domain name on host machine that maps to a web app in the container?  e.g. using sample-site.dev, instead of localhost:5000, from host machine to access the web app. 

Thank you.

**Comment by igor-lemon** [comment]
Hi @saun4app 
Local Host works in a different way. You can't use `sample-site.dev` if your web server was bounded to any another port different from 80 and 443 port. If you bound container with a WEB server to `5000` port you can use only sample-site.dev:5000. If you want to bind a container to 5000 port and use a hostname without port, you need to up some proxy. But I think that isn't flexible and we solve to use a solution like that: 
![image](https://user-images.githubusercontent.com/14764799/30180292-189304ba-9418-11e7-96c9-e982123d86fd.png)
![image](https://user-images.githubusercontent.com/14764799/30180338-3ded79b6-9418-11e7-9e2d-0d1e11b33f62.png)

**Comment by saun4app** [comment]
@igor-lemon, 

Thank you very much for your prompt and detailed reply.  Sorry, I was not being clear.  I am not particular about using a port number.  

Mainly, I looking for a way to bind domain (host) name, from the host machine, to a web app in a container.  Using your screen as an example, the ability to access the web apps from a browser on the host machine using `wordpress.dev`, `drupal.dev`. 

Basically, something similar to the feature from `nanobox`.  Please see `ADD A LOCAL DNS` section in https://guides.nanobox.io/php/slim/from-scratch/ and https://docs.nanobox.io/cli/dns/.

**Comment by igor-lemon** [comment]
So If I understand this is what I wrote above. But we have chosen a more simple way.
I don't know how the NanoBox made that app on 8080 port was opened by strong url on macOS, because on macOS and Windows you can't communicate directly with a container (by IP) only Linux provides full support.
If you want we can speak in Slack: [dockstation.slack.com](dockstation.slack.com)

**Comment by saun4app** [comment]
@igor-lemon, that is ok.  Thank for your help.

**Comment by igor-lemon** [comment]
You're welcome.
